tools/testing_utils/experiments/experiment_tests.csv

```diff
@@ -135,12 +142,17 @@ testing_archive.test_intt_not_num::intt must be a number
 testing_archive.test_beam_angle_dne::Slice must specify beam_angle that must be a list of numbers \(ints or floats\) which are angles of degrees off boresight \(positive E of N\)
 testing_archive.test_beam_angle_not_list::Slice must specify beam_angle that must be a list of numbers \(ints or floats\) which are angles of degrees off boresight \(positive E of N\)
 testing_archive.test_beam_angle_not_num::.*Slice must specify beam_angle that must be a list of numbers \(ints or floats\) which are angles of degrees off boresight \(positive E of N\)
-testing_archive.test_beam_order_dne::Slice must specify beam_order that must be a list of ints or lists \(of ints\) corresponding to the order of the angles in the beam_angle list
-testing_archive.test_beam_order_not_list::Slice must specify beam_order that must be a list of ints or lists \(of ints\) corresponding to the order of the angles in the beam_angle list
-testing_archive.test_beam_order_not_ints_or_lists::Slice must specify beam_order that must be a list of ints or lists \(of ints\) corresponding to the order of the angles in the beam_angle list
-testing_archive.test_beam_order_lists_not_ints::Slice must specify beam_order that must be a list of ints or lists \(of ints\) corresponding to the order of the angles in the beam_angle list
+testing_archive.test_beam_order_dne::Slice must specify rx_beam_order that must be a list of ints or lists \(of ints\) corresponding to the order of the angles in the beam_angle list
+testing_archive.test_beam_order_not_list::Slice must specify rx_beam_order that must be a list of ints or lists \(of ints\) corresponding to the order of the angles in the beam_angle list
+testing_archive.test_beam_order_not_ints_or_lists::Slice must specify rx_beam_order that must be a list of ints or lists \(of ints\) corresponding to the order of the angles in the beam_angle list
+testing_archive.test_beam_order_lists_not_ints::Slice must specify rx_beam_order that must be a list of ints or lists \(of ints\) corresponding to the order of the angles in the beam_angle list
 testing_archive.test_beam_order_lists_bad_index::Beam number .* could not index in beam_angle list of length .*. Slice: .*
 testing_archive.test_beam_order_bad_index::Beam number .* could not index in beam_angle list of length .*. Slice: .*
+testing_archive.test_tx_beam_order_dne::tx_beam_order must be a list of ints corresponding to the order of the angles in the beam_angle list. Slice: .*
+testing_archive.test_tx_beam_order_not_list::tx_beam_order must be a list of ints corresponding to the order of the angles in the beam_angle list. Slice: .*
+testing_archive.test_tx_beam_order_list_not_ints::tx_beam_order must be a list of ints corresponding to the order of the angles in the beam_angle list. Slice: .*
+testing_archive.test_tx_beam_order_listnum_too_high::Beam number .* in tx_beam_order could not index in beam_angle list of length .*. Slice: .*
```

Comment:
duplicate test name as above ^ "test_tx_beam_order_listnum_too_high"
maybe this doesn't matter though...

Reply:
The unit tests all pass with either one of the duplicate lines commented out, or with both included as they currently are. Frankly I expected that one of them would fail since they don't have the same error message.

Reply:
This is very strange